### PR TITLE
[NUI] Add Comment on the View.VisibilityChanged event

### PR DESCRIPTION
**Pull request by bshsqa**
### API Changes ###

 - ACR:

### CODE REVIEW

**Review by hinohie — APPROVED** [review]

File: src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs [hunk 1]
```diff
@@ -417,8 +417,18 @@ public partial class View
 
         /// <summary>
         /// An event for visibility change which can be used to subscribe or unsubscribe the event handler.<br />
-        /// This signal is emitted when the visible property of this or a parent view is changed.<br />
+        /// This event is sent when the visibility of this or a parent view is changed.<br />
         /// </summary>
+        /// <remarks>
+        /// <para>
+        /// If VisibilityChangedEventArgs.Type is SELF, then VisibilityChangedEventArgs.Visibility is true if this View's Visibility property is true.
```
Comment (Seoyeon2Kim) [inline src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs:424]: This sentence is a little awkward, especially on the grammar.
-> If A, B if C.
You mean, 
```suggestion
        /// If VisibilityChangedEventArgs.Type is SELF and  this View's Visibility property is true, then VisibilityChangedEventArgs.Visibility is true.
```
Is it what you want to explained?

File: src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs [hunk 2]
```diff
@@ -417,8 +417,18 @@ public partial class View
...
+        /// If VisibilityChangedEventArgs.Type is PARENT, VisibilityChangedEventArgs.Visibility is true if a parent's Visibility property has changed to true.
+        /// </para>
+        /// <para>
+        /// This event is NOT sent if the view becomes transparent (or the reverse), it's ONLY linked with View.Show() and View.Hide().
+        /// For reference, a view is only shown if it and it's parents (up to the root view) are also visible, are not transparent, and this view has a non-zero size.
```
Comment (Seoyeon2Kim) [inline src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs:429]:
```suggestion
        /// For reference, a view is only shown if the view and its parents (up to the root view) are also visible, they are not transparent, and the view has a non-zero size.
```

**Review by Seoyeon2Kim — CHANGES_REQUESTED** [review]
And please update the description of this PR

**Review by dongsug-song — APPROVED** [review]
approved

**Review by Seoyeon2Kim — APPROVED** [review]

**Pull request merged**